src/llm_tracekit/litellm/instrumentor.py

```diff
@@ -35,19 +35,20 @@ def __init__(
         application_name: Optional[str] = None,
         subsystem_name: Optional[str] = None,
     ):
+        self._custom_handler: Optional[LitellmCallback] = None
```

Comment:
you can init this in this function instead of waiting for `_instrument`, right after calling `generate_exporter_config` do `self._custom_handler = ...`, and then there is no need to store `self._config`